pkg/internal/imetrics/imetrics.go

```diff
@@ -7,7 +7,8 @@ import (
 
 // Config options for the different metrics exporters
 type Config struct {
-	Prometheus PrometheusConfig `yaml:"prometheus,omitempty"`
+	Prometheus  PrometheusConfig `yaml:"prometheus,omitempty"`
+	OTELMetrics bool             `yaml:"otel_metrics,omitempty" env:"BEYLA_INTERNAL_METRICS_OTEL_METRICS"`
```

Comment:
Maybe to be less redundant, rename YAML property to `otel` or `use_otel` and the env var to `BEYLA_INTERNAL_METRICS_OTEL` or `BEYLA_INTERNAL_METRICS_USE_OTEL`?

Even, to rely less on booleans for config, replace this property for something like `protocol` or `exporter`, which can be `none`, `prometheus` or `otel`.

Reply:
> Even, to rely less on booleans for config, replace this property for something like `protocol` or `exporter`, which can be `none`, `prometheus` or `otel`.

Is not a bad idea, but that would imply breaking changes, no? basically we have to force everyone to set `exporter` and `prometheus` config for internal metrics.

Reply:
I guess we can document that breaking change in Beyla 2.0.

If we want to keep backwards compatibility, maybe we could default it to `prometheus`, and explain that it will only have effect if the `prometheus` subsection is set.

But anyway that was just a suggestion. I'm fine with current implementation.